### ISSUE TITLE
[Bug] Edit pop cover is not centered and the filename input overlaps the full path

### ISSUE THREAD

**Issue by taki-sahli** [rommapp/romm#1295]
**RomM version**
Version 3.5.1

**Describe the bug**
The cover is not centered, and if the rom filename is long, it overlaps the path and become hard to edit.

**To Reproduce**
Edit rom with long name.

**Expected behavior**
Clear to see the filename of the rom with the path.

**Screenshots**
![before](https://github.com/user-attachments/assets/4e77855e-b0fe-41ac-a052-d6a86cd90b7c)

**Desktop:**

- OS: Windows 11
- Browser: Firefox
- Version: 132

**Additional context**
As a workaround, I use `Stylus` with this `css`:
```css
#file-name-label {
  left:   1rem !important;
  bottom: 0rem !important;
}

.cover {
  min-height: unset !important;
  max-height: unset !important;
}
```
This is the result:
![after](https://github.com/user-attachments/assets/24f12b4b-5b24-4ea8-889a-cb46623df2f5)

**Comment by zurdi15** [comment]
Partially fixed in the last 3.6.0 release. Still need to fix the overlap between name and path

**Comment by zurdi15** [comment]
Path overlap fixed in #1324 

![image](https://github.com/user-attachments/assets/076a5e21-a273-4cbc-bf28-02fe6138bb9b)

![image](https://github.com/user-attachments/assets/17e814e7-f8d8-49b3-86c4-9fd3a72e5fd1)
